Modules/BookList.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable */
+import Book from './BookClass.js';
```

Comment:
- [ ] Kindly remove the `eslint-disable` statement in your code. Eslint helps us to write clean, readable and industry-standard codes. Using `eslint-disable` statements defeats this purpose. 👍